DS30.ipynb

```diff
@@ -149,8 +149,14 @@
     "def nlargest(n, word_scores):\n",
     "    return heapq.nlargest(n, word_scores, key=lambda x: x[1])\n",
     "\n",
+    "def remove_punctuation(text):\n",
+    "    # hardcoded to avoid importing any library\n",
+    "    exclude = '!\"#$%&\\'()*+,-./:;<=>?@[\\\\]^_`{|}~0123456789'\n",
```

Comment:
Ok. I was just thinking on extracting plain english words to make a nice wordcloud or histogram (though I agree some symbols do have a sentiment attached to it).